commit 094ed50726a28d4797e16742a1c1fd9c3f578d20
Author: Jernej Skrabec <jernej.skrabec@siol.net>
Date:   Tue Jan 8 20:18:40 2019 +0100

    ARM: dts: sun8i: h3: Add ethernet0 alias to Beelink X2
    
    [ Upstream commit cc4bddade114b696ab27c1a77cfc7040151306da ]
    
    Because "ethernet0" alias is missing, U-Boot doesn't generate board
    specific MAC address. Effect of this is random MAC address every boot
    and thus new IP address is assigned to the board.
    
    Fix this by adding alias.
    
    Fixes: 7389172fc3ed ("ARM: dts: sun8i: h3: Enable dwmac-sun8i on the Beelink X2")
    Signed-off-by: Jernej Skrabec <jernej.skrabec@siol.net>
    [Maxime: Removed unneeded comment]
    Signed-off-by: Maxime Ripard <maxime.ripard@bootlin.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/sun8i-h3-beelink-x2.dts b/arch/arm/boot/dts/sun8i-h3-beelink-x2.dts
index 5d23667dc2d2..25540b7694d5 100644
--- a/arch/arm/boot/dts/sun8i-h3-beelink-x2.dts
+++ b/arch/arm/boot/dts/sun8i-h3-beelink-x2.dts
@@ -53,7 +53,7 @@
 
 	aliases {
 		serial0 = &uart0;
-		/* ethernet0 is the H3 emac, defined in sun8i-h3.dtsi */
+		ethernet0 = &emac;
 		ethernet1 = &sdiowifi;
 	};
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1063_094ed50726a28d4797e16742a1c1fd9c3f578d20.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1063_094ed50726a28d4797e16742a1c1fd9c3f578d20.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
